.tekton/determine-image-tag-task.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: determine-image-tag
+  namespace: rh-acs-tenant
+spec:
+  description: Determining image tag with custom logic.
```

Comment:
```suggestion
  description: Determines image tag with custom logic.
```

Reply:
After seeing the description show up in the pipeline, I have a desire for better
![image](https://github.com/stackrox/stackrox/assets/537715/c8651433-c835-4bc0-996a-818f6493d03d)

How about this?
> Determines the tag for the output image using the StackRox convention (from `make tag` output).

Reply:
fixed - https://github.com/stackrox/stackrox/pull/10749/commits/d124162924490686bc76f7a3d5fef97f860af434#diff-4fb9379bf71bb227cba8211a3ebc5a3914856b3dc08b02fc311f274ba9bec32eR8.